.github/CODEOWNERS

```diff
@@ -1,106 +1,586 @@
 # Instructions for CODEOWNERS file format and automatic build failure notifications:
 # https://github.com/Azure/azure-sdk/blob/master/docs/policies/opensource.md#codeowners
 
-###########
+#########
 # SDK
-###########
+#########
 
 # Catch all
 /sdk/                                           @AlexGhiondea
 
-# Core
+# ######## Core Libraries ########
+
 # PRLabel: %Azure.Core
 /sdk/core/                                      @pakrym @KrzysztofCwalina
 
+# PRLabel: %Azure.Identity
+/sdk/identity/                                  @schaabs @alexanderSher
+
 # Extensions
 /sdk/extensions/                                @pakrym
 
-# Compute
-# PRLabel: %Compute
-/sdk/compute/                                   @bilaakpan-ms @sandido @dkulkarni-ms @haagha @madewithsmiles @MS-syh2qs @grizzlytheodore
+# PRLabel: %EngSys
+/sdk/template/                                  @Azure/azure-sdk-eng
+
+# Smoke tests
+/common/SmokeTests/                             @AlexGhiondea @schaabs @heaths @tg-msft @jsquire
+
+
+# ######## Management Plane ########
+
+/**/*Management*/                               @allenjzhang @m-nash @markcowl @YalinLi0312 @bquantump @nisha-bhatia
```

Comment:
This one is broad and catches things like changes to `sdk/servicebus/Azure.Messaging.ServiceBus/tests/Administration/ServiceBusManagementClientLiveTests.cs`.   I don't have a good suggestion for tuning it, but maybe you do?

Reply:
I will leave that as-is for now... this particular one is only used for the PR owner suggestions.